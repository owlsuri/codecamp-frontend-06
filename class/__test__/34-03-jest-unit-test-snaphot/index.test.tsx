// 하나하나 테스트할 수 없다보니 똑같은 내용을 카피해서 변경되면 변경사항이 맞는지 확인 알림 보냄
import JestUnitTestSnapPage from '../../pages/34-03-jest-unit-test-snapshot' 
import { render } from "@testing-library/react"

it("컴포넌트가 기존이랑 바뀐게 없는지 비교해보자-스냅샷테스트", ()=>{
    const result = render(<JestUnitTestSnapPage />)
    expect(result.container).toMatchSnapshot()  
})