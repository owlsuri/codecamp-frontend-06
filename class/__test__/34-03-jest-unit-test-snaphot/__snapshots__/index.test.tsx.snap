// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`컴포넌트가 기존이랑 바뀐게 없는지 비교해보자-스냅샷테스트 1`] = `
<div>
  <div>
    <div>
      철수는 13살입니다.
    </div>
    철수취미입력 : 
    <input
      type="text"
    />
    <button>
      철수랑 놀러가기
    </button>
  </div>
</div>
`;
